sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -260,6 +279,35 @@ def _set_partition_key(self, value):
             self.message.annotations = annotations
             self.message.header = header
 
+    def try_add(self, event_data):
```

Comment:
Where is this function used?

Reply:
The **try_add** function is part of the API and supposed to be used by customers not by us.